qctrlopencontrols/dynamic_decoupling_sequences/predefined.py

```diff
@@ -830,34 +986,59 @@ def new_quadratic_sequence(
 def new_x_concatenated_sequence(
     duration=1.0, concatenation_order=None, pre_post_rotation=False, **kwargs
 ):
-    """
-    Creates X-Concatenated Dynamic Decoupling Sequence.
-    Concatenation of base sequence C(\tau/2)XC(\tau/2)X
+    r"""
+    Creates the :math:`X`-concatenated sequence.
 
     Parameters
     ----------
     duration : float, optional
-        defaults to None
-        The total duration of the sequence
+        Defaults to None.
+        The total duration of the sequence :math:`\tau`.
     concatenation_order : int, optional
-        defaults to None
-        The number of concatenation of base sequence
+        Defaults to None.
+        The number of concatenation of base sequence.
     pre_post_rotation : bool, optional
-        If True, a :math:`\\pi.2` rotation is added at the
+        If True, a :math:`X_{\pi/2}` rotation is added at the
         start and end of the sequence.
     kwargs : dict
-        Additional keywords required by
-        qctrlopencontrols.dynamical_decoupling_sequences.DynamicDecouplingSequence
+        Additional keywords required by DynamicDecouplingSequence.
 
     Returns
     -------
-    qctrlopencontrols.dynamical_decoupling_sequences.DynamicDecouplingSequence
-        X concatenated sequence
+    DynamicDecouplingSequence
+        The :math:`X`-concatenated sequence.
 
     Raises
     ------
     ArgumentsValueError
         Raised when an argument is invalid.
+
+    See Also
+    --------
+    new_xy_concatenated_sequence
+
+    Notes
+    -----
+    The :math:`X`-concatenated sequence is constructed by recursively concatenating
+    control sequence structures. It's parameterized by the concatenation order :math:`l` and
+    the duration of the total sequence :math:`\tau`. Let the :math:`l`-th order of concatenation
+    be denoted as :math:`C_l(\tau)`. In this scheme, zeroth order concatenation of duration
+    :math:`\tau` is defined as free evolution over a period of :math:`\tau`. We use a notation
+    of :math:`{\mathcal 1}(\tau)` to mean free evolution over duration :math:`\tau`.
+    We define the base sequence to be:
```

Comment:
```suggestion
    :math:`\tau` is defined as free evolution over a period of :math:`\tau`. Using the notation
    :math:`{\mathcal 1}(\tau)` to represent free evolution over duration :math:`\tau`, the
    the base sequence is:
```
The Microsoft Style Guide recommends avoiding "we" if possible (https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/we), so here's a suggestion (feel free to change to something equivalent)